src/components/FindMovie/FindMovie.js

```diff
@@ -1,55 +1,117 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './FindMovie.scss';
 
+import { PropTypes } from 'prop-types';
+import { MovieType } from '../types';
+
+import { getMovie } from '../../api/api';
 import { MovieCard } from '../MovieCard';
-import movies from '../../api/movies.json';
-
-export const FindMovie = () => (
-  <>
-    <form className="find-movie">
-      <div className="field">
-        <label className="label" htmlFor="movie-title">
-          Movie title
-        </label>
-
-        <div className="control">
-          <input
-            type="text"
-            id="movie-title"
-            placeholder="Enter a title to search"
-            className="input is-danger"
-          />
-        </div>
 
-        <p className="help is-danger">
-          Can&apos;t find a movie with such a title
-        </p>
-      </div>
+export const FindMovie = ({ addMovie }) => {
+  const [movieTitle, setMovie] = useState('');
```

Comment:
```suggestion
  const [title, setTitle] = useState('');
```
Якщо ти вже в компонеті Movie то зрозуміло що title це про фільм, не треба всюди це повторювати